src/services/EcomMLAppService.js

```diff
@@ -0,0 +1,10 @@
+
+const baseURL = 'https://us-central1-ecom-mercado-livre.cloudfunctions.net/app'
+
+const getAuthUrl = () => {
+  return `${baseURL}/mercado-livre/callback?store_id=${localStorage.getItem('store_id')}`
```

Comment:
Mais correto aqui seria usar o [`@ecomplus/utils`](https://developers.e-com.plus/utils/$ecomConfig.html) ou o `EcomAuth` pra identificar a loja:

```suggestion
  return `${baseURL}/mercado-livre/callback?store_id=${$ecomConfig.get('store_id')}`
```

Com `localStorage` você pode pegar o ID de outra loja se o administrador tiver aberto mais de um painel ao mesmo tempo (em abas diferentes do navegador).